### PR TITLE
feat: update all dependencies

### PR DESCRIPTION
**Pull request by dignifiedquire**

### CODE REVIEW

File: test/api/block.spec.js [hunk 1]
```diff
@@ -71,6 +71,7 @@ describe('.block', () => {
               done()
             })
         })
+        .catch(done)
```
Comment (diasdavid) [inline test/api/block.spec.js]: mocha3?

**Comment by diasdavid** [comment]
LGTM :)

File: test/api/block.spec.js (hunk 1 above)
Comment (dignifiedquire) [inline test/api/block.spec.js]: yes those changes are to make mocha happy :)

File: package.json [hunk 2]
```diff
@@ -30,14 +30,14 @@
     "url": "https://github.com/ipfs/js-ipfs-api"
   },
   "devDependencies": {
-    "aegir": "^3.2.0",
+    "aegir": "^5.0.1",
     "chai": "^3.5.0",
     "gulp": "^3.9.1",
     "interface-ipfs-core": "^0.4.3",
```
Comment (diasdavid) [inline package.json]: wanna update this one too?

Comment (dignifiedquire) [inline package.json]: greenkeeper already made a PR will merge that afterwards all the things are green

**Pull request merged**